libs/governance/src/lib/proposals-queries/proposals-queries.ts

```diff
@@ -0,0 +1,136 @@
+import { gql, useQuery } from '@apollo/client';
+import type { Proposal, ProposalVariables } from './__generated__/Proposal';
+import type {
+  ProposalsConnection,
+  ProposalsConnection_proposalsConnection_edges_node as ProposalNode,
+} from './__generated__/ProposalsConnection';
+
+const PROPOSALS_FRAGMENT = gql`
```

Comment:
chore (blocking): I don't like the idea of having a shared queries in the app. A lot of the value proposition of GQL is that we request only what we need, rather than everything every time. If we start sharing queries and 1/2 pages need a new field we will be requesting the field for every single page and doing nothing with it. I would rather colocate these queries, so that in each place we request exactly what is needed. Alternatively, if there is a use case for sharing the component that renders the proposals list then a lib that exposes that component would be good, as the data that is used is always guaranteed.